src/NServiceBus.Core.Tests/Msmq/MsmqQueueCreatorTests.cs

```diff
@@ -171,6 +171,23 @@ public void Should_blow_up_if_name_is_null()
             Assert.Throws<ArgumentNullException>(() => creator.CreateQueueIfNecessary(bindings, WindowsIdentity.GetCurrent().Name));
         }
 
+        [Test]
+        public void Verify_msmq_permissions_and_delete_has_lower_lenght_limit_compared_to_queue_name()
+        {
+            //increase to 50 and create will work but SetPermission and Delete will blow up
```

Comment:
@timbussmann @ramonsmits this is why the tests where failing, create was ok but since we no longer swallow setpermission issues this has now surfaced